internal/datastore/postgres/migrations/zz_migration.0020_add_watch_api_index.go

```diff
@@ -2,20 +2,17 @@ package migrations
 
 import (
 	"context"
-	"fmt"
 
 	"github.com/jackc/pgx/v5"
 )
 
-const addWatchAPIIndexToRelationTupleTable = `CREATE INDEX CONCURRENTLY IF NOT EXISTS ix_watch_index ON relation_tuple (created_xid);`
+const watchAPIIndexToRelationTupleTable = `
+	ON relation_tuple (created_xid);`
```

Comment:
`ON` seems [required as per the docs](https://www.postgresql.org/docs/current/sql-createindex.html), seems off to force callsite of `createIndexconcurrently` to provide it.

Reply:
Changed